src/dev/code_coverage/ingest_coverage/ingest.js

```diff
@@ -0,0 +1,103 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+const { Client } = require('@elastic/elasticsearch');
+import { createFailError } from '@kbn/dev-utils';
+import chalk from 'chalk';
+import { green, always } from './utils';
+import { fromNullable } from './either';
+import { COVERAGE_INDEX, TOTALS_INDEX } from './constants';
+
+const node = process.env.ES_HOST || 'http://localhost:9200';
+const redacted = redact(node);
```

Comment:
locally this looks okay, but its existence means i'm going to extreme scrutiny (sorry!).  e.g. redact will cover basic, but the function name is general

i'd _really_ like to see this handled outside of the repo, whether through a local node connected to the stats cluster or behind a vlan and ip whitelist.

@brianseeders do you know if we have options here?

Reply:
Great idea, but this is handled via vault

Reply:
https://github.com/elastic/kibana/blob/e635be1784873db91495ec17c6a411b33aa59bc5/vars/kibanaCoverage.groovy#L121

Reply:
Perhaps I'm missing something?

Reply:
no you got it, i mostly don't want to see vault data bubbling into kibana and environment variables.  it's not wrong, but just ups the scrutiny needed.

Reply:
and by environment variables i mean lower level sets.  we do have globals but keeping the surface area down as much as possible